airbyte-ci/connectors/connectors_insights/src/connectors_insights/insights.py

```diff
@@ -24,9 +24,12 @@
 
 
 def get_manifest_inferred_insights(connector: Connector) -> dict:
+    if connector.manifest_path is None or not connector.manifest_path.exists():
+        return {}
+
     manifest = connector.manifest_path.read_text()
 
-    schemas_directory = connector.code_path / connector.technical_name.replace("-", "_") / "schemas"
+    schemas_directory = connector.code_directory / connector.technical_name.replace("-", "_") / "schemas"
```

Comment:
Wait up, did I mess it up and it _never_ worked? How did I manage that without failing a test? Or does `code_path` exist, but does a different thing? What's the difference between them?